test/EndToEnd/tests/ServicesTest.ps1

```diff
@@ -654,12 +654,12 @@ function Test-InstallPackageAPIInstalledLowerVersionPackage
     $p = New-ClassLibrary
 
     # Act
-    [API.Test.InternalAPITestHook]::InstallPackageApi("microsoft.owin","2.0.0")
-    [API.Test.InternalAPITestHook]::InstallPackageApi("microsoft.owin","3.0.0")
+    [API.Test.InternalAPITestHook]::InstallPackageApi("nuget.librarymodel","6.2.0")
```

Comment:
We can't use `microsoft.owin` package any, because all versions except [last 4.2.2 version](https://www.nuget.org/packages/Microsoft.Owin/4.2.2) has vulnerabilities. but we need 2 versions for this test. So, I replaced the package with something doesn't have any vulnerabilities.